greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const nameUtil = require('./lib/namer.js');
+
+module.exports = exports = {};
+
+//process.argv is an array, hence why we use the index of the array later
```

Comment:
No points lot FOR NOW, but just a heads up that we'll be docking points for uncommented code and stray console.logs AND comments. Try to manage a branch of your own where you can have all your own notes on there and then submit a clean branch to Canvas.